### PR TITLE
HTS-1080 Updated README with endpoints

### PR DESCRIPTION
**Pull request by Jackie1988**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,5 +11,65 @@ and go to `PreviewRAML` on the left. In the box to enter the URL put in `http://
 If everything is OK you should see the api documentation. Otherwise, it should tell you what's wrong with the documentation
 
 
+### Endpoints
+
+All endpoints in this API are user-restricted. All calls to the API start with localhost:7004/individuals/help-to-save/
```
Comment (AndyHWChung) [inline README.md]: All _local_ calls

File: README.md [hunk 2]
```diff
@@ -11,5 +11,65 @@ and go to `PreviewRAML` on the left. In the box to enter the URL put in `http://
 If everything is OK you should see the api documentation. Otherwise, it should tell you what's wrong with the documentation
 
 
+### Endpoints
+
+All endpoints in this API are user-restricted. All calls to the API start with localhost:7004/individuals/help-to-save/ 
+
+We can use help-to-save-test-admin-frontend to build requests to hit these endpoints.
```
Comment (AndyHWChung) [inline README.md]: ...hit these endpoints directly _to the microservice locally or through the API platform in other environments_?

File: README.md [hunk 3]
```diff
@@ -11,5 +11,65 @@ and go to `PreviewRAML` on the left. In the box to enter the URL put in `http://
...
+
+## POST /account
+
+Sends a create account request to NS&I if the user is eligible. This endpoint requires no parameters.
```
Comment (AndyHWChung) [inline README.md]: The endpoint does require a JSON body however

File: README.md [hunk 4]
```diff
@@ -11,5 +11,65 @@ and go to `PreviewRAML` on the left. In the box to enter the URL put in `http://
...
+
+For example:
+ ```
+ /account
+ ```
+If the call is successful, expect a `201` response. If call is not successful expect a `400` 
+if there was a validation error or expect a `500` if there was a backend error.
```
Comment (AndyHWChung) [inline README.md]: `409` is returned when the account already exists

File: README.md [hunk 5]
```diff
@@ -11,5 +11,65 @@ and go to `PreviewRAML` on the left. In the box to enter the URL put in `http://
...
+ ```
+ /account
+ ```
+If the call is successful, expect a `200` response with JSON containing the eligibility result.
+If the given nino doesn't match that found in Auth expect a `403` or if the request validation 
+fails expect a `400` or if for another reason the call is not successful expect a `500` response.
+
+### Version
+
+Sign up for an HMRC developer hub account at {baseUrl}/developer/registration to
```
Comment (AndyHWChung) [inline README.md]: you'd also need our team to whitelist their application in order for them to see the documentation

File: README.md (hunk 1 above)
Comment (Jackie1988) [reply README.md]: done, thanks

File: README.md (hunk 2 above)
Comment (Jackie1988) [reply README.md]: It is possible to curl the endpoints in the API locally?

Comment (Jackie1988) [reply README.md]: ok thanks

File: README.md (hunk 5 above)
Comment (Jackie1988) [reply README.md]: ok added, thanks

**Comment by Jackie1988** [comment]
I have removed all mention of ```This endpoint requires no parameters```

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**